src/plugins/orangehrmRecruitmentPlugin/Api/CandidateInterviewSchedulingAPI.php

```diff
@@ -316,7 +313,7 @@ public function getValidationRuleForGetOne(): ParamRuleCollection
             ),
             new ParamRule(
                 self::PARAMETER_INTERVIEW_ID,
-                new Rule(Rules::IN_ACCESSIBLE_ENTITY_ID, [Interview::class])
+                new Rule(Rules::POSITIVE)
```

Comment:
hope you covered scenarios in  integration testing for changed param rules
in `src/plugins/orangehrmRecruitmentPlugin/Api/CandidateMarkingInterviewFailAPI.php` & `src/plugins/orangehrmRecruitmentPlugin/Api/CandidateInterviewSchedulingAPI.php`